BLE Indoor Positioning/src/main/java/com/nexenio/bleindoorpositioning/IndoorPositioning.java

```diff
@@ -29,10 +29,9 @@
     public static final long UPDATE_INTERVAL_MEDIUM = 500;
     public static final long UPDATE_INTERVAL_SLOW = 3000;
 
-    public static final double NO_SPEED_FILTER = -1;
+    public static final double _NOT_SET = -1;
```

Comment:
When I commented

    Rename to `MAXIMUM_MOVEMENT_SPEED_NONE` or `_NOT_SET` or `_INFINITE`

what I meant was

    Rename to `MAXIMUM_MOVEMENT_SPEED_NONE` or `MAXIMUM_MOVEMENT_SPEED_NOT_SET` or `MAXIMUM_MOVEMENT_SPEED_INFINITE`